javascript/frameworks/cap/src/sensitive-exposure/SensitiveExposureHeuristicSource.md

```diff
@@ -0,0 +1,30 @@
+# CAP Insertion of Sensitive Information into Log File
+
+If sensitive information is written to a log entry using the CAP Node.js logging API, a malicious user may be able to gain access to user data.
+
+Data that may expose system information such as full path names, system information, usernames and passwords should not be logged.
+
+## Recommendation
+
+CAP applications should not log sensitive information.
```

Comment:
I think it'd be great to add some examples of sensitive information and ways to mitigate it.

Reply:
sure, added a bit more of the details/list of examples of types of data, as from the out of the box description of what is [covered](https://github.com/github/codeql/blob/main/javascript/ql/src/Security/CWE-312/CleartextLogging.qhelp#L12)
the out of the box JS qhelp was a bit sparse on mitigation, but I think the recommendation in the [go one](https://codeql.github.com/codeql-query-help/go/go-clear-text-logging/) applies here too, so I have used similar wording to that help file

good suggestion!